pulsar-functions/runtime-all/pom.xml

```diff
@@ -48,7 +61,7 @@
 
     <dependency>
       <groupId>${project.groupId}</groupId>
-      <artifactId>pulsar-client-original</artifactId>
```

Comment:
I believe that is it better to have the full Pulsar Client.

We should forbid the NAR file to contain Pulsar code.
because it would introduce compatibility problems, data corruption or data loss